src/docs/product/data-management-settings/event-grouping/stack-trace-rules.mdx

```diff
@@ -31,6 +31,8 @@ stack.abs_path:**/node_modules/**         -app
 stack.abs_path:**/generated/**.js         -group
 ```
 
+-`discover {tabTitle:Enhancement Rules} -stack.abs_path:**/node_modules/** -group -stack.abs_path:**/app/utils/requestError.jsx -group -stack.abs_path:**src/getsentry/src/getsentry/** +app - -family:native max-frames=3 - -stack.function:fetchSavedSearches v-group -stack.abs_path:**/app/views/**.jsx stack.function:fetchData ^-group - -family:native stack.function:SpawnThread v-app -app -family:native stack.function:_NSRaiseError ^-group -app -family:native stack.function:std::* -app -family:native stack.function:core::* -app -`
```

Comment:
I don't think that will render nicely

Reply:
Whoops. I restored this example [here](https://github.com/getsentry/sentry-docs/pull/3916/files#diff-cb119bc9e50b510a3eecb3b5571682d0194d93837050f5da33bc2abf41c5ef8cR175), this must be a leftover.